workspaces/liferay-ups-workspace/client-extensions/liferay-ups-commerce-shipping-engine/src/main/resources/application-default.properties

```diff
@@ -1,3 +1,11 @@
+#
+# LXC
+#
+
+com.liferay.lxc.dxp.domains=localhost:8080
+com.liferay.lxc.dxp.mainDomain=localhost:8080
+com.liferay.lxc.dxp.server.protocol=http
```

Comment:
these lines should not be sent to upstream, please remove them

Reply:
I saw there are in master under a couple of CX like Adyen, Stripe, etc.